input/number/reports.md

```diff
@@ -0,0 +1,28 @@
+## This page should be used for casual reporting of issues.
+
+For help on formatting, see: 
+- [How to put multiple lines in a Markdown table cell (multiline table)](https://alvinalexander.com/text/how-put-multiple-lines-markdown-table-cell-multiline-table)
+- [Markdown table generator](https://www.tablesgenerator.com/markdown_tables)
+
+### <a name="valid-issues">Valid issue types:</a> 
+- Inconsistency with other similar input elements
+- Non-extensibly (functionality)
+- Non-extensibly (styling)
+- Lacks built-in accessibility 
+
+### Structure of a report
+
+#### Please use a <a href="#valid-issues">valid issue type</a>.
+
+##### At least one link is mandatory. 
+
+|  An example report                                                                                                                    |
+|---------------------------------------------------------------------------------------------------------------------|
+| <p><strong>Issue</strong>: `<valid issue type>`<br><strong>Links</strong>:<br>- `<link1>`<br>- `<link2>`<br>- `<link3>`<br><strong>Description</strong>:`<some description>`</p> |
+
+
+---
+
+|  Reports                                                                                                                    |
+|---------------------------------------------------------------------------------------------------------------------|
+| <p><strong>Issue</strong>: `Inconsistency with other similar input elements`<br> <strong>Links</strong>:<br>- [Input type “number” won't resize](https://stackoverflow.com/questions/22709792/input-type-number-wont-resize)<br>- [Why doesn't the <input> element respect min-width?](https://stackoverflow.com/questions/29470676/why-doesnt-the-input-element-respect-min-width)<br><strong>Description</strong>: Can't use the same CSS to handle inputs type='number' as the others.</p> |
```

Comment:
Adjust this to known issues not reports.

Reply:
As per above, not sure it is an issue yet. You commented below that it's per spec, which I'll ask about below, but feel like I should be able to drop something here that bothers me in working with forms.

Or if there were a different space?